### ISSUE TITLE
about tutorial.ipynb

### ISSUE THREAD

**Issue by Yuan-Jinghui** [pchlenski/manify#23]
Hi Philippe,

Long time no see! I noticed that the current tutorial is named `tutorial.ipynb` and introduces the clustering section. I think some simple modifications could improve it:

1. `accuracy = (y_pred == y_clustering).float().mean().item()`
   Although accuracy is also a commonly used metric for clustering, it requires some conversion, because clustering results don’t naturally align with the metric. For example, if `y_clustering = [0,0,0,1,1,1,2,2,2]` and the clustering result is `[1,1,1,2,2,2,0,0,0]`, we would still consider it accurate. A simple alternative would be using the `nmi` (Normalized Mutual Information) score from sklearn.

2. `X_clustering, y_clustering = pm.gaussian_mixture(num_points=1000, num_classes=4, seed=2025, task="classification")`
   For unsupervised learning, the generated noise seems too large. A better option might be:
   `X_clustering, y_clustering = pm.gaussian_mixture(num_points=1000, num_classes=4, seed=2025, task="classification", cov_scale_points=.1)`

In summary, I believe the suitable code should look like this:

```python
from sklearn.metrics import normalized_mutual_info_score
pm = manify.ProductManifold(signature=[(0, 16),(1,16),(-1,16)])

# Use classification labels, which identify clusters by their center
X_clustering, y_clustering = pm.gaussian_mixture(num_points=1000, num_classes=4, seed=2025, task="classification", cov_scale_points=.1)

# The RFK algorithm is essentially a sklearn-styled clustering algorithm, so we call it like this:
rfk = manify.RiemannianFuzzyKMeans(pm=pm, n_clusters=4, random_state=2025)
rfk.fit(X_clustering)
y_pred = rfk.predict(X_clustering)
nmi = normalized_mutual_info_score(y_clustering, y_pred)
print(f"Riemannian Fuzzy K-Means nmi: {nmi:.2f}")
```
Wish you all the best!

**Comment by pchlenski** [comment]
Hi Jungui, thanks for pointing this out. I've fixed this in https://github.com/pchlenski/manify/commit/5f1df6056e8b4f11c09fd1033fbf5d4cad3d0493 - please verify it is suitable.
